tests/bson/bson-toPHP-007.phpt

```diff
@@ -0,0 +1,122 @@
+--TEST--
+MongoDB\BSON\toPHP(): fieldPath typemaps without server
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
```

Comment:
Server-less tests shouldn't import this (see other BSON tests), as it ultimately requires `PHONGO-SERVERS.json` to exist. The extension check in `basic-skipif.inc` isn't very helpful anyway.